skl2onnx/common/_apply_operation.py

```diff
@@ -221,21 +221,23 @@ def apply_reciprocal(scope, input_name, output_name, container, operator_name=No
 
 
 def apply_reshape(scope, input_name, output_name, container, operator_name=None, desired_shape=None):
-    if len(list(i for i in desired_shape if i < 0)) > 1:
-        raise ValueError('There can only be one -1 in the targeted shape of a Reshape but got %s' % desired_shape)
 
     name = _create_name_or_use_existing_one(scope, 'Reshape', operator_name)
 
     if container.target_opset < 7:
```

Comment:
Should this be container.target_opset < 5 since reshape was most recently updated in opset 5?

Reply:
It is taken from onnxmltools. I assume it was for a good reason. The build set up on travis was trying other version of ONNX. We should do that again.